code/drasil-example/Drasil/SWHS/IMods.hs

```diff
@@ -210,25 +204,23 @@ eBalanceOnPCM :: InstanceModel
 eBalanceOnPCM = im eBalanceOnPCM_rc [qw temp_melt_P, qw time_final, qw temp_init, qw pcm_SA,
  qw pcm_HTC, qw pcm_mass, qw htCap_S_P, qw htCap_L_P]
   [sy temp_init $< sy temp_melt_P] (qw temp_PCM)
-   [0 $< sy time $< sy time_final] [makeCite koothoor2013] eBalanceOnPCMDeriv 
-   "eBalanceOnPCM" [balPCMDesc_note]
+   [0 $<= sy time $<= sy time_final] [makeCite koothoor2013] eBalanceOnPCMDeriv 
+   "eBalanceOnPCM" balPCMDescNotes
 
 eBalanceOnPCM_rc :: RelationConcept
 eBalanceOnPCM_rc = makeRC "eBalanceOnPCM_rc" (nounPhraseSP
   "Energy Balance on PCM to find temperature of PCM")
   balPCMDesc balPCM_Rel -- eBalanceOnPCML
```

Comment:
Here, a completely different `Sentence` from the Notes is passed in. Should `RelationConcept` be taking a `Sentence` here? @JacquesCarette

Reply:
I don't actually know... I'm not sure what either are supposed to mean, and where they are used.